Wireframe/css/style.css

```diff
@@ -0,0 +1,7 @@
+body {
+    background-color: turquoise;
+}
+
+header {
+    text-align: center;
```

Comment:
It doesn't seem like you have laid out your web page in the way the wireframe shows yet. Would you like some help with the layout?

Reply:
Thank you, I have made some changes to the layout and submitted pull requests.  I am not sure how to add needs review label?